README.md

```diff
@@ -186,6 +186,32 @@ Update [values.yaml](https://github.com/openclarity/apiclarity/blob/master/chart
 
 4. Open APIClarity UI in the browser: <http://localhost:8080/>
 
+## Database
+
+SQLite, for local testing, and PostgreSQL databases are supported.
+
+To choose your database type, use the `DATABASE_DRIVER` environment variable:
+
+* LOCAL: Select SQLite, and create a local file called `db.db`
```

Comment:
are you sure there is a  need to create the db.db file? I think it is generated automatically

Reply:
I meant: "When LOCAL is selected, it creates a file called 'db.db'", I was not saying you need to create it.
I'll rephrase it.